CHANGELOG.md

```diff
@@ -15,6 +15,9 @@ about writing changes to this log.
 - [TVIST1-706](https://jira.itkdev.dk/browse/TVIST1-706):
   Added checkbox giving the option to not send receipt upon
   creating hearing post response
+- [TVIST1-673](https://jira.itkdev.dk/browse/TVIST1-673):
+  Allowed configuration of mail template custom fields to
+  either text or textArea.
```

Comment:
I would write “textarea” (all lowercase).

Reply:
Updated in 81896df1e4a3900c171dacc9dbf863168bf41171.